tests/README.md

```diff
@@ -0,0 +1,92 @@
+named `delayed_config.yaml`).
+`config.yaml` consists of test configurations, k8s configurations and SM-client arguments which are being passed to the
+node on activation.
+The k8s configuration part of the config file will be concatenated to the k8s object files that are used by the test.
+
+One important variable in the config files is the `genesis_delta`, this is the representation in seconds of the time
+since deploying bootstrap node until the miners start mining, which means that this time should be sufficient for all
+nodes to be ready.
+If not ready by this time the test will fail.
+
+#### Running Automation Locally
+
+Notice: Before running automation you need to have permissions to our GCP account management.
+There is a document named 'instructions' in 1password with detailed instructions on how to setup all requirements for 
+automation. 
+
+There are two options for running automation locally:
+
+* running pytest
+```
+python3 -m pytest -s -v path/to/your_test.py --tc-file=path/to/your_test_config.py --tc-format=yaml
+```
+this will run the local python automation code with the compiled miners `develop` image.
+
+* running make command, for example:
+```
+make dockertest-p2p-elk
+``` 
+when running a make command a new compiled image of the local branch will be created and run by the automation code.
+
+#### Logging
+Logs can be viewed by accessing the local ELK cluster while the test is running.
+Kibana's IP can be resolved by a kubectl command: `kubectl get services -n YOUR_NAMESPACE`
+Logs will be dumped to a main ES server in two cases:
+* the test run has failed
+* "is_dump: True" was added to the test config file
+
+In any other case the logs will be gone for good.
+
+#### Logical Test Objects Separation
+When starting a new test a namespace will be created for that test and that test only.
+All deployments, statefulset and other objects will be created under that namespace and that namepsace only.
+
+#### ELK Cluster
+Automation ELK is built from Fluent-bit, Logstash, Kibana and Elasticsearch.
+We use fluent-bit as a shipper for the logs from SM-client to Logstash and modify those logs there before moving them to ES.
+After setting the ELK cluster we wait for it to be ready, this process might take a couple of minutes.
+
+#### Node pool
+Node pools provide a physical test objects separation in oppose to namespacing.
+When ELK is ready we add the node pool that will serve the Spacemesh pods. This node pool is necessary in order to get all of the resources needed (only for the SM-clients) in advance so our deployment will happen in a shorter and a more predictable time.
+The process of creating/deleting/altering a node pool is blocking since only one action can be made simultaneously. This means that if someone just started a test and wants to create a node pool he will have to wait for the completion of the previously started action.
```

Comment:
```suggestion
The process of creating/deleting/altering a node pool is blocking since only one action can be made simultaneously. This means that if someone just started a test and wants to create a node pool they will have to wait for the completion of the previously started action.
```

Reply:
fixed